release-notes/3.0/preview/api-diff/preview9/.Net/standalone-packages/3.0-preview9-standalone-packages.md

```diff
@@ -0,0 +1,6 @@
+# API Difference netcoreapp3.0-preview8 vs netcoreapp3.0-preview9
```

Comment:
nit: Consistency with https://github.com/dotnet/core/blob/master/release-notes/3.0/preview/api-diff/preview7/standalone-packages/3.0-preview7-standalone-packages.md#api-difference-standalone-preview6-vs-standalone-preview7

Update the title?
```suggestion
# API Difference standalone-preview8 vs standalone-preview9
```
Or maybe:
```suggestion
# API Difference standalone-3.0-preview8 vs standalone-3.0-preview9
```

Also, if it is empty, is it even worth including? We could just remove this all together (and the link).